<?xml version="1.0" ?>
<testsuite errors="0" failures="0" name="django.contrib.auth.tests.auth_backends.NoBackendsTest" tests="1" time="0.001">
	<testcase classname="django.contrib.auth.tests.auth_backends.NoBackendsTest" name="test_raises_exception" time="0.001"/>
	<system-out/>
	<system-err/>
</testsuite>
